### ISSUE TITLE
Brokers logo is needed...

### ISSUE THREAD

**Issue by imougy** [tradingticket/TradeItIosTicketSDKLib#24]
Please add icon for the brokers in the UITableView of the login.  It makes people feel better when they see the logo of their broker.

**Comment by acegreen** [comment]
@imougy @TheAntonioReyes I would suggest maybe using a UICollectionView with each cell having a nice image of the broker plus the name. Also a tiny disclaimer at the bottom to say "We will not share any of the information etc." on the login screen itself.

**Comment by acegreen** [comment]
@TheAntonioReyes I could work something for those 2 views if you would like

**Comment by imougy** [comment]
@acegreen I like the disclaimer line! A simple UITableView would be fine with the image on the side as some broker's name can take up the full-screen width.
Thanks!

**Comment by acegreen** [comment]
the simple UITableViewController is fine but looks meh on iPad in landscape. Just a suggestion maybe for future redesigns. 

The more important matter would be the scrolling on login page getting stuck.

![photo 2015-12-23 1 11 06 pm](https://cloud.githubusercontent.com/assets/10794609/11982234/57a95c18-a977-11e5-814a-4c6fec4347bf.png)

**Comment by imougy** [comment]
@acegreen, I agree on both, iPad development is pain in the ass and no one is using it!

**Comment by TheAntonioReyes** [comment]
We'll add in the disclaimer for future versions. In terms of the using the logos our legal team is still working closely with brokers to get permission to use their logos. We have permission from some but not most, so at this time it doesn't really make sense to add them in. We'll continue to work on this and update once this become feasible.

Thanks

**Comment by acegreen** [comment]
Ok thats fine.

Can we also add a VERY IMPORTANT disclaimer at the final order page, where it says "Estimated Cost"

We need to clearly say that this is their own TYPICAL broker fees. Otherwise some might awesome you guys are charging on top of the broker fees. A one liner would be perfect
